products/sle/main.pm

```diff
@@ -787,23 +781,25 @@ elsif (get_var('XFSTESTS')) {
         loadtest 'kernel/change_kernel';
     }
     prepare_target;
-    if (get_var('XFSTESTS_SKIP_INSTALL', 0) || check_var('XFSTESTS', 'installation') || is_pvm || check_var('ARCH', 's390x')) {
+    if (get_var('XFSTESTS_INSTALL', 0) || check_var('XFSTESTS', 'installation') || is_pvm || check_var('ARCH', 's390x')) {
```

Comment:
Seems here should be check_var('XFSTESTS_INSTALL', 1)

Reply:
IMO, it's equal to get_var('XFSTESTS_INSTALL', 0), but get_var set default value 0